src/OpenLoco/src/World/CompanyAi.h

```diff
@@ -89,8 +89,7 @@ namespace OpenLoco
         Station stations[4]; // 0x06 0x4AE Will lists stations created that vehicles will route to
         uint8_t trackObjId;  // 0x3E 0x4E6 track or road (with high bit set)
         uint8_t pad_3F;
-        uint8_t mods; // 0x40 0x4E8 track or road
-        uint8_t pad_41;
+        uint16_t mods;        // 0x40 0x4E8 track or road
```

Comment:
is this 2 bytes for "track and road" or is this one byte for track and one byte for road?

Reply:
It's two bytes used for the mod types could be road mods or a track mods depending on the track type.